pages/support.md

```diff
@@ -4,6 +4,20 @@
 }
 </script>
 
+## Version Support Status
```

Comment:
I don't know what this means or how to parse it. I suggest "Supported versions" instead, without title case.